jams/eval.py

```diff
@@ -32,6 +32,9 @@ def validate_annotation(ann, namespace):
     '''Validate that the annotation has the correct namespace,
```

Comment:
Well, just so that the function name reflects the functionality, I'd say yes (`coerce_and_validate`)